src/routes/join/+page.svelte

```diff
@@ -21,19 +21,9 @@
 	<form
 		class="flex flex-col items-center text-center max-w-md w-full p-8 bg-white shadow-md rounded-lg"
 		method="POST"
-		action="?/join"
+		action={data.user?.role != 'Student' ? '?/joinAnon' : '?/joinStudent'}
```

Comment:
10x developer move at gøre det her og så lave to forskellige schemas til hver action 
![image](https://github.com/user-attachments/assets/d5db48eb-4d95-4fdb-92ba-1eddf6b01fd6)